service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -267,6 +272,7 @@ public DynamicList buildRespondentDynamicList() {
     private final List<Element<SentDocuments>> documentsSentToParties;
 
     @JsonIgnore
+    @Deprecated
     public List<Element<Applicant>> getAllApplicants() {
         return applicants != null ? applicants : new ArrayList<>();
     }
```

Comment:
What was the justification for this to be marked as Deprecated? @tomaszpowroznik

Reply:
The story is about replacing Applicant and Solicitor with new LocalAuthority model. Applicant and Solicitor are kept for backward compatibility purpose (legacy cases on prod). After data migration could be removed.